kedro-datasets/kedro_datasets_experimental/rioxarray/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""``AbstractDataset`` implementation to load/save data from/to a geospatial raster files."""
+from __future__ import annotations
+
+from typing import Any
+
+import lazy_loader as lazy
+
+# https://github.com/pylint-dev/pylint/issues/4300#issuecomment-1043601901
+GeotiffDataset: Any
```

Comment:
Could we do `GeoTIFFDataset` here? Following convention I did for `NetCDF`: https://github.com/kedro-org/kedro-plugins/blob/main/kedro-datasets/kedro_datasets/netcdf/netcdf_dataset.py.

Especially since the formal name is GeoTIFF. https://en.wikipedia.org/wiki/GeoTIFF

Reply:
Yes would be better.